integration/tests/cook/util.py

```diff
@@ -1635,12 +1645,11 @@ def _get_compute_cluster_factory_fn():
 
 
 def using_kubernetes():
-    return get_kubernetes_compute_cluster() is not None
+    return os.getenv("COOK_TEST_COMPUTE_CLUSTER_TYPE", "mesos") == "kubernetes"
 
 
 def using_mesos():
-    return _get_compute_cluster_factory_fn() == 'cook.mesos.mesos-compute-cluster/factory-fn'
-
+    return os.getenv("COOK_TEST_COMPUTE_CLUSTER_TYPE", "mesos") == "mesos"
```

Comment:
Seems like we should extract this into a function:

```
def compute_cluster_type_under_test():
    return os.getenv("COOK_TEST_COMPUTE_CLUSTER_TYPE", "mesos")
````

Reply:
Good idea.